node/src/accountant/test_utils.rs

```diff
@@ -352,11 +443,18 @@ impl AccountantBuilder {
                 .make_result(ReceivableDaoMock::new())
                 .make_result(ReceivableDaoMock::new()),
         );
-        let pending_payable_dao_factory = self.pending_payable_dao_factory_opt.unwrap_or(
-            PendingPayableDaoFactoryMock::new()
-                .make_result(PendingPayableDaoMock::new())
-                .make_result(PendingPayableDaoMock::new())
-                .make_result(PendingPayableDaoMock::new()),
+        let sent_payable_dao_factory = self.sent_payable_dao_factory_opt.unwrap_or(
+            SentPayableDaoFactoryMock::new()
+                .make_result(SentPayableDaoMock::new())
+                .make_result(SentPayableDaoMock::new())
+                .make_result(SentPayableDaoMock::new()),
+        );
+        let failed_tx_dao_for_pending_payable_scanner =
+            FailedPayableDaoMock::new().retrieve_txs_result(vec![]);
```

Comment:
I don't know why this is here, it looks finshy, please try to use directly in the test(s).